google/cloud/testing_util/BUILD.bazel

```diff
@@ -44,22 +43,12 @@ cc_library(
     name = "google_cloud_cpp_testing",
     srcs = google_cloud_cpp_testing_srcs,
     hdrs = google_cloud_cpp_testing_hdrs,
-    defines = selects.with_or({
-        (
-            "@bazel_tools//src/conditions:linux_x86_64",
-            "@bazel_tools//src/conditions:linux_s390x",
-            "@bazel_tools//src/conditions:linux_ppc64le",
-            "@bazel_tools//src/conditions:linux_ppc",
-            "@bazel_tools//src/conditions:linux_aarch64",
-        ): [
+    defines = select({
+        "@platforms//os:linux": [
             "GOOGLE_CLOUD_CPP_HAVE_GETRUSAGE",
             "GOOGLE_CLOUD_CPP_HAVE_RUSAGE_THREAD",
         ],
-        (
-            "@bazel_tools//src/conditions:darwin",
-            "@bazel_tools//src/conditions:freebsd",
-            "@bazel_tools//src/conditions:openbsd",
-        ): [
+        "@platforms//os:macos": [
```

Comment:
As I am sure you know, macOS != FreeBSD and != OpenBSD.  So this is not strictly preserving the previous behavior.  OTOH, I am not sure the previous approach worked. 🤷   I think we should make this change and fix it if it is a problem.

Reply:
Yes, I realize this is not preserving behavior. But I wasn't sure why that behavior was needed or why the special cases for freebsd and openbsd were added. We don't claim to support those platforms, so I'm not sure if any of it actually works. I'm happy to add those cases back if we want.

Reply:
Sorry to be unclear, I think we should merge as-is.